osgp/shared/osgp-ws-smartmetering/src/main/resources/bindings/sm-common.xjb

```diff
@@ -0,0 +1,21 @@
+<!--
+
+    Copyright 2015 Smart Society Services B.V.
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<jxb:bindings
+  version="2.0"
+  xmlns:jxb="http://java.sun.com/xml/ns/jaxb"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <jxb:bindings schemaLocation="../schemas/common-ws-smartmetering.xsd">
```

Comment:
"../schemas/common.xsd"  ?

Reply:
Nope, "../schemas/common-ws-smartmetering.xsd" is correct